src/components/ProvinceDropdown/ProvinceDropdown.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import Select from "material-ui/Select"
+import PropTypes from "prop-types"
+import { MenuItem } from "material-ui/Menu"
+import { InputLabel } from "material-ui/Input"
+import { FormControl, FormHelperText } from "material-ui/Form"
+import ProvinceDropdownData from "./province.data"
+
+export default class ProvinceDropdown extends React.Component {
+  state = {
+    province: "",
+  }
+
+  handleProvinceChange = event => {
+    this.setState({ province: event.target.value }, () => {
+      this.props.onchange(this.state.province)
```

Comment:
Personally, I would store the province in a variable, then pass it through setState and onChange.

```
handleProvinceChange = event => {
  const province = event.target.value
  
  this.setState({province})
  this.props.onChange(province)
}
```

Since we're using an extra variable, you don't have to use the setState callback to make sure state will be updated in time.